tests/csrf_tests/tests.py

```diff
@@ -185,10 +185,21 @@ def test_csrf_token_in_header(self):
         Check that we can pass in the token in a header instead of in the form
         """
         req = self._get_POST_csrf_cookie_request()
-        req.META['HTTP_X_CSRFTOKEN'] = self._csrf_id
+        req.META[settings.CSRF_HEADER_NAME] = self._csrf_id
```

Comment:
I'm not sure this needs to be changed.

Testing the default value seems useful.